tests/openassetio/test/manager/resources/suite_executeSuiteTests.py

```diff
@@ -0,0 +1,82 @@
+#
+#   Copyright 2013-2022 The Foundry Visionmongers Ltd
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+Test cases that are run as part of the unit tests for `executeSuite`.
+They assert the expected public API of `FixtureAugmentedTestCase`, and
+that the harness implementation supplies the correct state to each test.
+"""
+
+# pylint: disable=invalid-name
+# pylint: disable=missing-class-docstring,missing-function-docstring
+
+from openassetio.hostAPI import Manager, Session
+from openassetio.test.manager.harness import FixtureAugmentedTestCase
+
+
+__all__ = []
+
+
+class Test_executeSuite_session(FixtureAugmentedTestCase):
+
+    def test_when_called_then_session_is_set(self):
+        self.assertIsInstance(self._session, Session)
+
+    def test_when_called_then_session_has_expected_host(self):
+        self.assertEqual(
+            self._session.host().identifier(),
+            "org.openassetio.test.manager"
```

Comment:
[consider] Perhaps for clarity it would be worth naming the stub host identifier more explicitly, e.g. 
```suggestion
            "org.openassetio.test.manager.stubHost"
```

Reply:
Improved [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/184/commits/0bcc7baac69a49057d52b19d50d88ec782154eed). I didn't use 'stub' in the end - as technically, it felt like it was a real host - ie, the test harness is a totally legit host in its own right.